sktime/performance_metrics/forecasting/probabilistic/_classes.py

```diff
@@ -964,3 +964,72 @@ def get_test_params(cls, parameter_set="default"):
         params1 = {}
         params2 = {"multivariate": True}
         return [params1, params2]
+
+
+class AUCalibration(_BaseDistrForecastingMetric):
+    r"""Area under the calibration curve for distributional predictions.
+
+    Computes the unsigned area between the calibration curve and the diagonal.
+
+    The calibration curve is the cumulative curve of the sample of
+    predictive cumulative distibution functions evaluated at the true values.
+
+    Mathematically, let :math:`d_1, \dots, d_N` be the predictive distributions,
+    let :math:`y_1, \dots, y_N` be the true values, and let :math:`F_i` be the
+    cumulative distribution function of :math:`d_i`.
+
+    Define the calibration sample as :math:`c_i := F_i(y_i)`, for
+    :math:`i = 1, \dots, N`. For perfect predictions, the sample of :math:`c_i` will be
+    uniformly distributed on [0, 1], and i.i.d. from that uniform distribution.
+
+    Let :math:`c_{(i)}` be the :math:`i`-th order statistic of the sample of
```

Comment:
I am bit confused here... Are there two different meaning of `I`? 

I interpret the `i`'s in line 981 as time index. I.e. c_i is the sample at index i.
 However. the `i` here (line 985)  is the index that is describing the size of the element. I.e. c_i is the i'ths smallest element.

Reply:
The "classical statistics" convention is that, if you use the brackets in the subscript, i.e., $c_{(i)}$ instead of $c_i$, you get the "order statistics", i.e., the $i$-th smallest of the numbers in the sample $c_i, i=1\dots N$.

Reply:
Ok I have overseen the brackets....